content/blog/grpc-web-interceptor.md

```diff
@@ -17,16 +17,16 @@ Similar to other gRPC languages, gRPC-web supports _unary_ and
 server-_streaming_ interceptors. For each kind of interceptor, we've [defined an
 interface][interceptor.js] containing a single `intercept()` method:
 
-- `UnaryInterceptor`
-- `StreamInterceptor`
+* `UnaryInterceptor`
+* `StreamInterceptor`
```

Comment:
Can these markdown-prettifier changes be undone too -- i.e., preserve whatever list marker is used? (There are several instances of this change throughout this file.)

Reply:
Hi! Just finished a few fixups for this commit. After re-running formatting there were still quite a few side effects so i needed to take a couple passes doing some manual linting. I've disabled most of my tooling that usually runs in the background so there shouldn't be any more unexpected side effects. Sorry about that! The diffs should be much easier to parse through without all the extra churn as well.

Reply:
Thanks!